lib/MusicBrainz/Server/Entity/Preferences.pm

```diff
@@ -44,6 +44,13 @@ has [qw(email_on_no_vote email_on_notes email_on_vote)] => (
     lazy => 1,
 );
 
+has 'email_on_abstain' => (
+    isa => 'Bool',
+    default => sub { shift->email_on_vote },
+    is =>'rw',
```

Comment:
```suggestion
    is => 'rw',
```

Reply:
This is already the case in the existing preferences so amending all in an extra commit.